docs/UserGuide.md

```diff
@@ -115,30 +117,50 @@ Currently, there are 6 types of drinks that can be tracked by tCheck's sales tra
 
 
 #### 1.1 Updating the number of drinks sold : `s-update`
-Updates the number of drinks sold for each drink type as given in the user input.
+Updates the number of drinks sold for each drink type as given in the user input. The previous records will
+be overwritten.
 
+Format: `s-update DRINK [MORE_DRINKS]`
+* where `DRINK` is formatted as `A/NUM`
+    * `A` refers to the drink's abbreviation.
+    * `NUM` refers to the number of drinks sold. It should be a **non-negative unsigned integer** that is 
+less than or equal to 99999999.
+* The sales of at least one drink item should be recorded when using this command.
 
-Format: `s-update A/NUM B/NUM C/NUM ...`
-* `A`, `B`, `C` are abbreviations for the drink types.
-* `NUM` refers to the number of drinks sold. It should be a non-negative integer.
+Example: Let's say you want to update the sales of Brown Sugar Boba Milk, `BSBM`, to 100 
+and Brown Sugar Boba Black Tea, `BSBBT`, to 120 in the Sales Tracker. You can follow these instructions:
 
-Example:
-* `s-update BSBM/100 BSBBT/120` Updates the sales of Brown Sugar Boba Milk `BSBM` to 100 and
- Brown Sugar Boba Black Tea `BSBBT` to 120.
- 
- <div markdown="span" class="alert alert-primary">
- 
-:information_source: Note about the update: <br>
+1. Type `s-update BSBM/100 BSBBT/120` into the _Command Box_.
+2. Press "Enter" to execute the command.
 
-The record in tCheck will be overwritten by the input that you provide. <br>
- 
- </div>
+Outcome:
+
+1. The _Result Display_ will show a success message.
+2. You can now see that the sales of the two drinks have been updated accordingly in the Sales Tracker inside the
+_Main View_. 
+
+{Example outcome screenshot to be added later}
 
 #### 1.2 Listing the number of drinks sold : `s-list`
-Shows a list of all types of drinks sold in the Sales Tracker.
+Shows a list of all types of drinks sold in the Sales Tracker. The list of drinks shown is 
```

Comment:
I am not sure if we should include the UI part (as in where it will be shown on the GUI) in the first line for the command description?  If we include, should we say Sales Tracker inside the _Main View_?

Reply:
Like in s-update, you did not talk about GUI there, so I am wondering if it should be consistent.

Reply:
I think it should be clear as I mentioned Sales Tracker inside the Main View at the introduction of the sales-related features.

Reply:
I have talked about Sales Tracker in the introduction of the sales-related features. Would that be sufficient?